src/components/Dashboard/TalkDashboard.vue

```diff
@@ -220,9 +223,9 @@ function scroll({ direction }: { direction: 'backward' | 'forward' }) {
 				{{ t('spreed', 'Check devices') }}
 			</NcButton>
 		</div>
-		<div class="title">
+		<header class="title">
```

Comment:
Maybe h2-h5? Definitely not a header tag should be used, it's a semantic block, like main or footer

Reply:
Yes. `<header>` is a kind of banner landmark for all the heading elements. It must contain a direct heading element inside.

In this case, it should be `<h3>`:
1. Talk
2. Dashboard
3. Upcoming meetings